keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -275,20 +275,57 @@ with a metric.
 
 There could be a large volume of requests for a specific resource which might result in the identified apiserver being unable to serve the proxied requests. This scenario should not occur too frequently, since resource types which have large request volume should not be added or removed during an upgrade -- that would cause other problems, too.
 
-TODO: security / cert stuff.
+We should ensure at most one proxy, rather than proxying the request over and over again (if the source apiserver has an incorrect understanding of what the destination apiserver can serve).
+
+To prevent Man in the Middle attack we will not expose information about apiserver IP/endpoint and the trust bundle control (used to authenticate server while proxying) in REST APIs.
```

Comment:
The problem isn't "exposing" it there -- the trust bundle is already distributed to all clients -- the problem is permitting someone with api access to inject a malicious one.

Reply:
Modified, is it better now?

Reply:
yeah!